README.md

```diff
@@ -209,6 +217,20 @@ docker run -v "$(pwd):/home/dependabot/dependabot-script" -w /home/dependabot/de
 ```
 docker run --rm -v "$(pwd):/home/dependabot/dependabot-script" -w /home/dependabot/dependabot-script -e ENV_VARIABLE=value dependabot/dependabot-core bundle exec ruby ./generic-update-script.rb
 ```
+### GitHub Actions Standalone
+The easiest and most common way to run DependaBot on GitHub is using the built-in
+DependaBot service as described [here](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions). This is recommended for most users.
+
+However, sometimes you may need to run DependaBot manually either for testing, or to enable features/plugins that are
+not currently available in DependaBot. This is relatively straight-forward to achieve with a shell-based GitHub action.
+
+  * In your GitHub repository, create a directory `.github/workflows` if it doesn't already exist.
+  * Copy [manual-github-actions.yaml](./manual-github-actions.yaml) into that directory.
```

Comment:
The link mentioned at lines 228 and line 229 will be not be valid, once you merge this PR and delete your branch.